configs/unet/unet-s5-d16_fcn_4xb4-40k_refuge-256x256.py

```diff
@@ -0,0 +1,9 @@
+_base_ = [
+    '../_base_/models/fcn_unet_s5-d16.py', '../_base_/datasets/refuge.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_40k.py'
+]
+crop_size = (256, 256)
+data_preprocessor = dict(size=crop_size)
+model = dict(
+    data_preprocessor=data_preprocessor,
+    test_cfg=dict(crop_size=(256, 256), stride=(170, 170)))
```

Comment:
Should solve conflict between annotations (which has label 0,1,2), `num_classes=2` in config, and `reduce_zero_label=True` in config.